apps/sms/views/conversation/js/conversation.js

```diff
@@ -751,6 +785,13 @@ var ConversationView = {
     this.cleanFields();
     this.initRecipients();
     this.updateComposerHeader();
+
+    Compose.setupLock({
+      canSend: () => this.hasValidRecipients(),
+      // Null means that we want to use default type detection strategy.
+      getType: () => this.hasEmailRecipients() ? 'mms' : null
```

Comment:
could also be: `this.hasEmailRecipients() && 'mms'`; not sure which one is more readable :)